frontend/@types/console/index.d.ts

```diff
@@ -35,6 +35,7 @@ declare interface Window {
     prometheusBaseURL: string;
     prometheusTenancyBaseURL: string;
     quickStarts: string;
+    releaseVersion: string;
```

Comment:
Let's have the console operator wire this through to the backend using the `RELEASE_VERSION` env var. This would also let us show a version in the About dialog for normal users who can't see the `ClusterVersion` resource. cc @jhadvig